jannovar-core/src/main/java/de/charite/compbio/jannovar/reference/TranscriptModel.java

```diff
@@ -116,6 +137,24 @@ public String getGeneID() {
 	}
 
 	/**
+	 * Return mapping containing alternative gene IDs, as parsed from RefSeq GFF3 file
+	 * 
+	 * Popular alternative identifiers (used as keys here):
+	 * 
+	 * <dl>
+	 * <dt>GeneID</dt>
+	 * <dd>String of the Entrez/NCBI gene ID</dd>
+	 * <dt>Genbank</dt>
+	 * <dd>String of the Genbank accession</dd>
+	 * <dt>HGNC</dt>
+	 * <dd>String of the numeric HGNC (Human Gene Nomenclature Committee) identifier</dd>
+	 * </dl>
+	 */
+	public ImmutableSortedMap<String, String> getAltGeneIDs() {
```

Comment:
Year. but for example in exomiser we need always entrez-geneIDs.  So we must be sure that the correct name is used in the map to get it.